src/dev/build/tasks/clean_tasks.js

```diff
@@ -122,7 +125,7 @@ export const CleanExtraFilesFromModulesTask = {
       // scripts
       '**/*.sh',
       '**/*.bat',
-      '**/*.exe',
+      '**/[!ctags-win32]*.exe',
```

Comment:
Looks like this line didn't get removed in my PR 🤦‍♂

Reply:
nvm, I will update it

Reply:
Done.